tutorials/simple_time_series_example.ipynb

```diff
@@ -0,0 +1,406 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pandapipes import networks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the pandapipes network can be created/ loaded:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "net = pandapipes.networks.simple_gas_networks.gas_meshed_delta()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is the delta STANET network from the chapter [Meshed Networks](https://pandapipes.readthedocs.io/en/latest/networks/meshed/meshed_networks.html) with the medium gas."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"pics/time_series_simple_example_network.png\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Furthermore, the network can be prepared. For this purpose the time series for sinks and sources have to be defined. Since there is only one sink and one source, the respective csv file only consists of one column. The number of lines corresponds to the desired number of time steps, in this case 10."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"pics/time_series_simple_example_csv_files.png\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "profiles_sink = pd.read_csv(os.path.join(pp_dir, 'test', 'pipeflow_internals', 'data',\n",
+    "                                         'test_time_series_sink_profiles.csv'), index_col=0)\n",
+    "profiles_source = pd.read_csv(os.path.join(pp_dir, 'test', 'pipeflow_internals', 'data',\n",
+    "                                           'test_time_series_source_profiles.csv'), index_col=0)\n",
+    "ds_sink = DFData(profiles_sink)\n",
+    "ds_source = DFData(profiles_source)\n",
+    "\n",
+    "const_sink = control.ConstControl(net, element='sink', variable='mdot_kg_per_s',\n",
+    "                                  element_index=net.sink.index.values, data_source=ds_sink,\n",
+    "                                  profile_name=net.sink.index.values.astype(str))\n",
+    "const_source = control.ConstControl(net, element='source', variable='mdot_kg_per_s',\n",
+    "                                    element_index=net.source.index.values,\n",
+    "                                    data_source=ds_source,\n",
+    "                                    profile_name=net.source.index.values.astype(str))\n",
+    "del const_sink.initial_powerflow\n",
+    "const_sink.initial_pipeflow = False\n",
+    "del const_source.initial_powerflow\n",
+    "const_source.initial_pipeflow = False"
```

Comment:
Is it really necessary to do this "manually"? If yes, maybe the default settings should me modified in the future @SimonRubenDrauz

Reply:
it is still, yes. But I already modified it in pandapower from initial_powerflow to initial_run. Already created a PR and need to merge it. As soon as I merged that changes in pandapower, I will do the same in pandapipes. Then we can delete it.

Reply:
sounds good, thank you :-)